addons/website_slides/controllers/main.py

```diff
@@ -457,27 +459,77 @@ def _get_user_slide_authorization(self, slide_id):
             status = 'not_authorized'
         return {'status': status, 'slide': slide, 'channel_id': slide.sudo().channel_id.id}
 
+    @staticmethod
+    def _get_valid_invite_values(channel_id=False, invite_hash=False, partner_id=False):
```

Comment:
Nice! It could also be pushed below, as it is private and used for the route, it should not be read/shown/visible before these.